truss/remote/baseten/service.py

```diff
@@ -29,7 +29,12 @@ def is_live(self) -> bool:
     def is_ready(self) -> bool:
         raise NotImplementedError
 
-    def predict(self, model_request_body: Dict):
+    def predict(
+        self,
+        model_request_body: Dict,
+        model_id: Optional[str] = None,
```

Comment:
what's the point of adding `model_id` and model_version_id` here if they're unused?